frontend/components/Dashboard/BoardList/CardBody.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from "react";
+import Link from "next/link";
+import {
+  HoverCardTrigger,
+  HoverCardContent,
+  HoverCardArrow,
+  HoverCardRoot,
+} from "../../Primitives/HoverCard";
+import { BoardType } from "../../../types/board";
+import { ColumnType } from "../../../types/column";
+import Card from "../../Primitives/Card";
+import Flex from "../../Primitives/Flex";
+import CardHeader from "./CardHeader";
+import { styled } from "../../../stitches.config";
+import Shape from "../../Primitives/Shape";
+import Text from "../../Primitives/Text";
+import InputTitle from "../../Title/InputTitle";
+
+const Circle = styled(Shape, {
+  size: "$32",
+});
+
+const HandleCols = React.memo<{ columns: ColumnType[] }>(({ columns }) => {
+  return (
+    <>
+      {columns.map((column) => {
+        const trigger = (
+          <Circle
+            variant="circle"
+            align="center"
+            justify="center"
+            css={{
+              color: "white",
+              backgroundColor: column.color === undefined ? "gray" : column.color,
+            }}
+          >
+            {column.cardsOrder.length}
+          </Circle>
+        );
+        return (
+          <HoverCardRoot key={column._id}>
+            <HoverCardTrigger asChild>{trigger}</HoverCardTrigger>
+            <HoverCardContent sideOffset={0}>
+              {column.title}
+              <HoverCardArrow />
+            </HoverCardContent>
+          </HoverCardRoot>
+        );
+      })}
```

Comment:
Can the `column.color` field be undefined? I searched for it and I believe it's a string

```suggestion
      {columns.map((column) => (
          <HoverCardRoot key={column._id}>
            <HoverCardTrigger asChild>
            <Circle
               variant="circle"
               align="center"
               justify="center"
               css={{
                  color: "white",
                  backgroundColor: column.color || "gray",
               }}
            >
              {column.cardsOrder.length}
            </Circle>
            </HoverCardTrigger>
            <HoverCardContent sideOffset={0}>
              {column.title}
              <HoverCardArrow />
            </HoverCardContent>
          </HoverCardRoot>
        ))}
```